spec/src/main/asciidoc/method-query.asciidoc

```diff
@@ -151,152 +155,139 @@ Explanation of the BNF elements:
 
 === Query by Method Name Keywords
 
-The following tables list the _Query by Method Name_ keywords that must be supported by Jakarta Data providers, except where explicitly indicated for a type of database.
+An implementation of Query by Method Name must support the following types of operation.
 
-[cols="12,~,25"]
+.Query by Method Name Actions
+[cols="12,~"]
 |===
-| Action | Description | Not Required For
+| Action | Description
 
 |`find`
 |Returns entity instances representing the records which satisfy the restriction, or representing all records if there is no restriction.
-|Key-value, Wide-Column
 
 |`delete`
 |Deletes every record which satisfy the restriction, or all records if there is no restriction, and returns either no result (`void`) or the number of records deleted.
-|Key-value, Wide-Column
 
 |`count`
 |Returns the number of records which satisfy the restriction, or the total number of records if there is no restriction.
-|Key-value, Wide-Column
 
 |`exists`
 |Returns `true` if at least one record satisfies the restriction or if there is at least one record in the database when there is no restriction.
-|Key-value, Wide-Column
 |===
 
-[NOTE]
-====
-The "Not Required For" column indicates the database types for which the respective keyword is not required or applicable.
-====
-Jakarta Data implementations must support the following list of Query by Method Name keywords, except where indicated for a database type. A repository method must raise `java.lang.UnsupportedOperationException` or a more specific subclass of the exception if the database does not provide the requested functionality.
+An implementation of Query by Method Name must support the following keywords.
```

Comment:
Don't get rid of this. It is what allows for NoSQL implementations that are incapable of a subset of keywords, such as `Or`. The TCK relies on this for accurate testing.

Reply:
I didn't get rid of anything. I moved it.

Reply:
> Don't get rid of this. It is what allows for NoSQL implementations that are incapable of a subset of keywords, such as Or. 

FTR, `Or` is not on the list of things which are only supported by a subset of databases. It was listed as 

> Not required for: Key-value, Wide-Column

But so was `findBy`, so that wasn't a meaningful additional restriction.

I hope you see how the presentation in terms of tables was making this more confusing.

Reply:
> Not required for: Key-value, Wide-Column

Indeed, this was listed for _every single keyword_.